pkg/plugins/error.go

```diff
@@ -0,0 +1,21 @@
+package plugins
+
+const (
+	UNSIGNED ErrorCode = iota
+	MODIFIED
+	INVALID
```

Comment:
all three can be a single error code right?  and then we use the existing status for more details on the exact signature error.